Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mixed = np.zeros(r.size)\n",
+    "for i in range(mixed.size):\n",
+    "    if i < signal.size:\n",
+    "        mixed[i] = np.add(r[i], signal[i])\n",
+    "    else:\n",
+    "        mixed[i] = r[i]\n",
+    "plt.hist(mixed,bins = 450)\n",
+    "plt.title('Data After Signal Injection')\n",
+    "plt.xlabel('Measurement')\n",
+    "plt.ylabel('Counts')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The mean has moved from 250 to 350. The distance it has moved to the right, 100, is exactly the mean of the background distribution, and this is no coincidence. On average, the background produces 100 particle events, so if on average the signal, independent of the background, produces 250 particle events, then of course since both events are happening at the same time, the overall number of observed particles in an average signal event, when we take the background into account, will be the sum of the particles from the background plus the particles from the signal, which on average is going to be 250 + 100 = 350. So it makes perfect sense that the mean moved 100 units to the right for the signal. It retained its shape as a Poisson distribution because the sum of Poisson distributions is a Poisson distribution, naturally. The net effect of all of this is that the observed measurement is not a measurement of the signal, but a measurement of the signal plus background, so that  the observed value is more likely to be stronger than the true injected signal. Even though it's a poisson distribution, due to the central limit theorem, the signal looks gaussian. So for the large sample size I have, it seems symmetric. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 3a"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The plotted histogram shows the relationship for the probability of occurrence of any injected signal given an observed signal. The strongest relationship can be observed right along the diagonal for an injected signal to show up. Taking and analyzing a sample point it seems that the offset between the injected signal and the observed signal is 100. Which is the value of the mean of our poisson background distribution."
```

Comment:
Problem 3A:
- I think the explanation makes sense. You could add how the reverse relationship works for this 2D histogram, since we can look at either possible observed signal values given a true injected signal, or possible true injected values given an observation value. I think overall these 2D histograms show us the spread in both directions of possible values.
- Graphing background + injected signal makes sense.
- All parts of the graph look good. You could add a slightly more descriptive title if you wanted, but otherwise is fine as is.